sklearn/preprocessing/_label.py

```diff
@@ -86,12 +48,10 @@ def _encode(values, uniques=None, encode=False, check_unknown=True):
     ----------
     values : array
         Values to factorize or encode.
-    uniques : array, optional
-        If passed, uniques are not determined from passed values (this
+    uniques : array
+        Uniques are not determined from passed values (this
```

Comment:
the part in the parenthesis should be removed it seems?
I guess this could just be "The unique values in `values`"

Reply:
maybe we can also repeat that it should be sorted when relying on numpy